Geometry/GEMGeometryBuilder/src/ME0GeometryBuilder.cc

```diff
@@ -0,0 +1,544 @@
+    if (DDfetch(*is, numbOfPads))
+      nPads = numbOfPads.doubles()[0];
+  }
+  LogTrace("ME0GeometryBuilder") << ((nStrips == 0.) ? ("No nStrips found!!")
+                                                     : ("Number of strips: " + std::to_string(nStrips)));
+  LogTrace("ME0GeometryBuilder") << ((nPads == 0.) ? ("No nPads found!!")
+                                                   : ("Number of pads: " + std::to_string(nPads)));
+
+  // EtaPartition specific parameter (size)
+  std::vector<double> dpar = fv.logicalPart().solid().parameters();
+  double b = dpar[4] / cm;  // half bottom edge
+  double B = dpar[8] / cm;  // half top edge
+  double L = dpar[0] / cm;  // half apothem
+  double t = dpar[3] / cm;  // half thickness
+
+#ifdef EDM_ML_DEBUG
+  LogTrace("ME0GeometryBuilder") << " name of logical part = " << fv.logicalPart().name().name() << std::endl;
+  LogTrace("ME0GeometryBuilder") << " dpar is vector with size = " << dpar.size() << std::endl;
+  for (unsigned int i = 0; i < dpar.size(); ++i) {
+    LogTrace("ME0GeometryBuilder") << " dpar [" << i << "] = " << dpar[i] / 10 << " cm " << std::endl;
+  }
+  LogTrace("ME0GeometryBuilder") << "size  b: " << b << "cm, B: " << B << "cm,  L: " << L << "cm, t: " << t << "cm "
+                                 << std::endl;
+#endif
+
+  std::vector<float> pars;
+  pars.emplace_back(b);
+  pars.emplace_back(B);
+  pars.emplace_back(L);
+  pars.emplace_back(nStrips);
+  pars.emplace_back(nPads);
+
+  bool isOdd =
+      false;  // detId.chamber()%2; // this gives the opportunity (in future) to change the face of the chamber (electronics facing IP or electronics away from IP)
+  ME0BoundPlane surf(boundPlane(fv, new TrapezoidalPlaneBounds(b, B, L, t), isOdd));
+  std::string name = fv.logicalPart().name().name();
+  ME0EtaPartitionSpecs* e_p_specs = new ME0EtaPartitionSpecs(GeomDetEnumerators::ME0, name, pars);
+
+  ME0EtaPartition* etaPartition = new ME0EtaPartition(detId, surf, e_p_specs);
+  return etaPartition;
+}
+
+ME0GeometryBuilder::ME0BoundPlane ME0GeometryBuilder::boundPlane(const DDFilteredView& fv,
+                                                                 Bounds* bounds,
+                                                                 bool isOddChamber) const {
+  // extract the position
+  const DDTranslation& trans(fv.translation());
+  const Surface::PositionType posResult(float(trans.x() / cm), float(trans.y() / cm), float(trans.z() / cm));
+
+  // now the rotation
+  //  DDRotationMatrix tmp = fv.rotation();
+  // === DDD uses 'active' rotations - see CLHEP user guide ===
+  //     ORCA uses 'passive' rotation.
+  //     'active' and 'passive' rotations are inverse to each other
+  //  DDRotationMatrix tmp = fv.rotation();
+  const DDRotationMatrix& rotation = fv.rotation();  //REMOVED .Inverse();
+  DD3Vector x, y, z;
+  rotation.GetComponents(x, y, z);
+  // LogTrace("GEMGeometryBuilderFromDDD") << "translation: "<< fv.translation() << std::endl;
```

Comment:
delete commented-out code

Reply:
@kpedro88 : I'll delete all comments but these are comments of the old DDD part.